doc/reference/orm.rst

```diff
@@ -1,25 +1,53 @@
 :banner: banners/orm_api.jpg
 
-.. _reference/orm:
-
 =======
 ORM API
 =======
 
-Recordsets
-==========
+.. automodule:: odoo.models
+
+Creating Models
```

Comment:
This should be merged into Models.
If the structure suggested in a previous comment is adopted, it will fit just right.